<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1207</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MyOpenGLWidget" name="OpenGLWidget" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>991</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePath">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>1041</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Podaj ścieżke pliku/plików csv</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Load">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Załaduj</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sukces</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenFile">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybierz plik/pliki</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="G">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>991</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Generate">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generuj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushPrevious">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labelFrame">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>510</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushNext">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>260</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1207</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyOpenGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">myopenglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
